admin-dev/themes/new-theme/js/components/tinymce-editor.js

```diff
@@ -163,26 +162,14 @@ class TinyMCEEditor {
         const textareaLinkSelector = `.nav-item a[data-locale="${textareaLocale}"]`;
 
         $(textareaLinkSelector, tabContainer).on('shown.bs.tab', () => {
-          const form = $(textarea).closest('form');
           const editor = window.tinyMCE.get(textarea.id);
           if (editor) {
             // Reset content to force refresh of editor
             editor.setContent(editor.getContent());
           }
-
-          EventEmitter.emit('languageSelected', {
```

Comment:
@NeOMakinG I re-applied this https://github.com/PrestaShop/PrestaShop/commit/99cfdbf1fb8c7d50db1cf1ec0df9c2638ac2c018

Can you confirm I did it well ?

Reply:
Yes it looks fine